test/evergreen.yml

```diff
@@ -1264,6 +1264,22 @@ tasks:
             set -o verbose
             sh dist/s_clang-scan 2>&1
 
+  - name: directio
+    depends_on:
+    - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "compile wiredtiger"
```

Comment:
Jenkins job has the configure options `../configure --enable-diagnostic --enable-strict`. Not sure if other options (as part of the standard `compile wiredtiger` func will make any difference for the test output. Adding @ddanderson for comment.

Reply:
I think other things are just extras.